server/src/game/components/cult.rs

```diff
@@ -30,8 +30,8 @@ impl Cult{
         Cult::set_ordered_cultists(self, game);
     }
     pub fn on_any_death(mut self, game: &mut Game){
-        self.sacrifices_needed = self.sacrifices_needed.map(|s| s.saturating_sub(1));
-        if let Some(s) = self.sacrifices_needed{
+        self.sacrifices_required = self.sacrifices_required.map(|s| s.saturating_sub(1));
+        if let Some(s) = self.sacrifices_required{
```

Comment:
Bro didn't read the whole comment smh, call this captured value 'required' so you don't need to do 'required: s' later